### PR TITLE
Fix compability with Kinto 4.0 about unique fields (Kinto/kinto#763)

### PR DESCRIPTION
**Pull request by leplatrem**
@Natim r?

### CODE REVIEW

File: kinto_redis/storage.py [hunk 1]
```diff
@@ -138,13 +138,9 @@ def _bump_timestamp(self, collection_id, parent_id, record=None,
...
     @wrap_redis_error
     def create(self, collection_id, parent_id, record, id_generator=None,
-               unique_fields=None, id_field=DEFAULT_ID_FIELD,
+               id_field=DEFAULT_ID_FIELD,
                modified_field=DEFAULT_MODIFIED_FIELD,
                auth=None):
-        self.check_unicity(collection_id, parent_id, record,
-                           unique_fields=unique_fields, id_field=id_field,
-                           for_creation=True)
-
```
Comment (leplatrem) [inline kinto_redis/storage.py]: I could also have created an empty method for retro-compat...

Comment (Natim) [inline kinto_redis/storage.py]: We don't need retro-compat for this plugin since previous version are buil-in previous version of Kinto,

**Comment by Natim** [comment]
``RedisStorageTest.test_create_raise_unicity_error_if_provided_id_exists``

Apparently this test is still present in the MemoryStorageTest -> https://github.com/Kinto/kinto/blob/master/kinto/tests/core/test_storage.py#L246-L254

File: kinto_redis/storage.py (hunk 1 above)
Comment (Natim) [inline kinto_redis/storage.py]: The problem is that removing that call doesn't check for id unicity anymore.

**Pull request merged**